docs/sql-guide/functions/function-datetimeadd.md

```diff
@@ -0,0 +1,105 @@
+---
+title: DATETIMEADD()
+layout: default
+parent: SQL functions
+grand_parent: SQL guide
+---
+
+# DATETIMEADD() function
+
+The `DATETIMEADD()` function adds a time duration specified using time unit and integer value to a date-time and then returns the results.
+
+## Syntax
+
+```
+DATETIMEADD(timeunit, int_expr, date-time)
+```
+
+## Arguments
+
+| Argument | Data type | Description | Required | Further information |
+|---|---|---|---|---|
+| timeunit | string | String value that specifies the time unit for the duration to be added to the target date-time. | Yes | [Time units](#additional-information).|
+| int_expr | integer | Integer value specified as literal or expression to be added as time duration to the target date-time. | Yes | |
+| date-time | timestamp | The target date-time specified as literal or expression the time duration is added to. | Yes | |
+
+
+## Returns
+
+| Data type | Value |
+|---|---|
+| timestamp | Returns date-time added to the specified time duration. |
+
+## Additional information
+
+### timeunit
+
+{% include /sql-guide/datetimeadd-function-timeunit-table.md %}
+
+## Examples
+
+### Add 62 years to date of birth to derive retirement date. 
+Here DATETIMEADD() is used to do a popular date arithmetic.
```

Comment:
You can probably leave this out because your heading has explained what's going on.
Ultimately "popular" is a subjective viewpoint, and not guaranteed to be everyone's point of view :)